packages/logs/src/domain/configuration.ts

```diff
@@ -40,23 +39,24 @@ export function validateAndBuildLogsConfiguration(
     return
   }
 
+  const allowedConsoleApis = Object.keys(ConsoleApiName) as ConsoleApiName[]
   let forwardConsoleLogs: StatusType[] = []
   if (isExperimentalFeatureEnabled('forward-logs') && initConfiguration.forwardConsoleLogs !== undefined) {
     if (
       (initConfiguration.forwardConsoleLogs !== 'all' && !Array.isArray(initConfiguration.forwardConsoleLogs)) ||
       (Array.isArray(initConfiguration.forwardConsoleLogs) &&
-        initConfiguration.forwardConsoleLogs.some((api) => !includes(CONSOLE_APIS, api)))
+        initConfiguration.forwardConsoleLogs.some((api) => !includes(allowedConsoleApis, api)))
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏you could use `objectHasValue` to check this, see [example here](https://github.com/DataDog/browser-sdk/blob/main/packages/rum-core/src/domain/configuration.ts#L84)